util/pingpong.c

```diff
@@ -1439,6 +1439,11 @@ static int pp_getinfo(struct ct_pingpong *ct, struct fi_info *hints,
 		ct->rx_ctx_ptr = NULL;
 	}
 
+	if (hints && ((hints->caps & FI_DIRECTED_RECV) == 0)) {
+		(*info)->caps &= ~FI_DIRECTED_RECV;
+		(*info)->rx_attr->caps &= ~FI_DIRECTED_RECV;
+	}
```

Comment:
Why not just always clear this if the test doesn't use it?

Reply:
I was just being careful, that's all.

Reply:
I assumed.  I kept this as-is.  Thanks!